apps-rendering/package-lock.json

```diff
@@ -18632,7 +18632,7 @@
     "which-module": {
       "version": "2.0.0",
       "resolved": "https://registry.npmjs.org/which-module/-/which-module-2.0.0.tgz",
-      "integrity": "sha512-B+enWhmw6cjfVC7kS8Pj9pCrKSc5txArRyaYGe088shv/FGWH+0Rjx/xPgtsWfsUtS27FkP697E4DDhgrgoc0Q=="
+      "integrity": "sha1-2e8H3Od7mQK4o6j6SzHD4/fm6Ho="
     },
     "which-typed-array": {
       "version": "1.1.8",
```

Comment:
You may want to drop the changes to `package-lock.json`? I'm not sure they're related to this change?

Reply:
Grr this sha issues...

Reply:
Have removed the changes to package-lock by checking out the version on main